cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -2,7 +2,7 @@ import interceptGetRequests from '../helpers/interceptGetRequests';
 import getTotalPageSize from '../helpers/getTotalPageSize';
 import roundTo2Decimals from '../helpers/roundTo2Decimals';
 
-const MAX_PAGE_WEIGHT_KB = 50;
+const MAX_PAGE_WEIGHT_KB = 53;
```

Comment:
Can we keep this at 50, but add an override for the page type to the `PAGE_TYPE_PAGE_WEIGHT_MAPPING` object below?